DatadogSessionReplay/Sources/Feature/SessionReplayFeature.swift

```diff
@@ -67,8 +67,12 @@ internal class SessionReplayFeature: DatadogRemoteFeature {
         self.performanceOverride = PerformancePresetOverride(
             maxFileSize: UInt64(10).MB,
             maxObjectSize: UInt64(10).MB,
-            meanFileAge: 5, // equivalent of `batchSize: .small` - see `DatadogCore.PerformancePreset`
-            minUploadDelay: 1 // equivalent of `uploadFrequency: .frequent` - see `DatadogCore.PerformancePreset`
+            meanFileAge: 2, // vs 5s with `batchSize: .small` - see `DatadogCore.PerformancePreset`
+            uploadDelay: (
+                initial: 2, // vs 5s with `uploadFrequency: .frequent`
+                range: 0.6..<6, // vs 1s ..< 10s with `uploadFrequency: .frequent`
+                changeRate: 0.75 // vs 0.1 with `uploadFrequency: .frequent`
+            )
```

Comment:
💡 For insights how these numbers translate into actual timings, see: https://github.com/DataDog/dd-sdk-ios/blob/e1b3b7fea686c954c5584a57c779ab1eabaf847a/DatadogCore/Sources/Core/Upload/DataUploadDelay.swift#L17-L38